client/src/partials/fiscal/fiscal.new.js

```diff
@@ -0,0 +1,70 @@
+angular.module('bhima.controllers')
+.controller('FiscalCreateController', FiscalCreateController);
+FiscalCreateController.$inject = [
+    '$state', 'ScrollService', 'FiscalService', 'NotifyService', 'util'
+];
+
+/**
+ * This controller is responsible for creating a debtor group. It provides
+ * utility functions for submission and error handling.
+ *
+ * @todo refactor code to remove redundant features introduced previously
+ *
+ * @module fiscal/new
+ */
+function FiscalCreateController($state, ScrollTo, fiscalService, Notify, util) {
+  var vm = this;
+  // expose state for optional view elements
+  vm.state   = $state;
+  vm.fiscal  = {};
+  vm.endDate = endDate;
+
+  vm.maxLength = util.maxTextLength; 
+
+  function endDate(){
+    var start_date = new Date(vm.fiscal.start_date);
+    var previousDay;
+    vm.end_date = start_date.setMonth(start_date.getMonth() + vm.fiscal.number_of_months);
+  }
+
+  function handleException(error) {
+    // expose error to view
+    vm.exception = error;
+  }
+
+  settupDefaults();
+
+  function settupDefaults() {
+    vm.submit = submit;
+  }
+
+  function submit(fiscal) {    
+    var submitFiscal;
+    fiscal.$setSubmitted();
+
+    // ensure all Angular form validation checks have passed
+    if (fiscal.$invalid || !vm.fiscal.start_date) {
```

Comment:
Do you need to check `!vm.fiscal.start_date`?  It should be automatically required, and set the form to $invalid if it is not defined.